Doc/library/pathlib.rst

```diff
@@ -462,6 +462,20 @@ Pure paths provide the following methods and properties:
       True
 
 
+.. method:: PurePath.is_relative_to(*other)
+
+   Return whether or not this path is relative to the *other* path.
+
+      >>> from pathlib import PurePath
```

Comment:
Nit: can you remove the import? Other snippets don't show the import.

Reply:
What are the general guidelines for imports in interpreter examples? In the `pathlib` documentation, does anything from the `pathlib` module not require an import?

Reply:
I don't know if there are official guidelines (you may ask on the ML). Personally, I think imports in interpreter snippets are most of the time a distraction, especially when there are tens of snippets on a single documentation page.

Generally speaking, I expect the user to be able to make out that the `PurePath` class listed in the documentation for the `pathlib` module needs the corresponding import before being used.

Reply:
>Generally speaking, I expect the user to be able to make out that the PurePath class listed in the documentation for the pathlib module needs the corresponding import before being used.

@pitrou Thanks for the clarification, I think that's perfectly reasonable and agree that an excessive number of imports can be distracting. Unless someone else disagrees with your rationale, I don't think that a new ML thread is needed.

I think that the explicit imports are primarily appropriate when importing modules that are external to the documentation file. This can be see in one of the existing examples already in ``pathlib.rst``, when ``os`` is imported:
```
>>> import os
>>> p = PurePath('/etc')
>>> os.fspath(p)
'/etc'
```

Reply:
> I think that the explicit imports are primarily appropriate when importing modules that are external to the documentation file.

Agreed.

Reply:
guys, thanks :)